index.html

```diff
@@ -19,12 +19,18 @@
       <a class="name" href="https://github.com/Cluab">Ibrahim Al-Yousefi</a>
     </div>
     <nav>
-      <img class="toolbar"src="img/Union.svg" alt="side-bar">
+      <!-- <img class="toolbar"src="img/Union.svg" alt="bar"> -->
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+     </div>
```

Comment:
- [ ] I think it will be better if your mobile menu container height touches the top and the bottom of the viewport so that your UI can look good and also look like the Figma design template you chose 👍🏽 

You can make the container have `min-height: 100vh` 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194937174-a77d6558-f056-4703-9405-e1e14ee2b3f9.png)  | ![image](https://user-images.githubusercontent.com/63473584/194937554-f2277ea7-05c2-4487-8caf-c152698a402a.png)  |